handbook/product-design/product-design.rituals.yml

```
@@ -8,3 +8,91 @@
   autoIssue:   # Enables automation of GitHub issues
     labels: [ "#g-endpoint-ops" ] # label to be applied to issue
     repo: "confidential"
+-
+  task: "Design sprint review"
+  startedOn: "2024-03-07" 
+  frequency: "Triweekly" 
+  description: "After the last estimation session, the Head of Product Design reviews the board with each group designer and de-prioritizes all design issues that were not estimated. The Head of Product Design also collects all items that are product-driven and puts them in the 🎁🗣 Feature fest meeting agenda to consider for continuing work. The number of de-prioritized issues should be recorded in the KPI spreadsheet." 
```

```suggestion
  description: "Clear out the drafting board of all issues that are not estimated but leave the items we want to take in the next sprint on the drafting board. Record the number of dropped stories for KPIs (all user stories that did not meet the 3 week drafting timeline)." 
```

Updating this to reflect the calendar event description.